.github/workflows/config.yml

```diff
@@ -116,16 +116,21 @@ jobs:
           python3 test/update_yml_precomputed_files.py
 
       - name: Run run_analysis.rb
-        run: |          
+        run: |
           bundle exec rake workflow:analysis_tests
 
       - name: Upload precomputed buildstocks
+        if: ${{ always() }}
```

Comment:
Even if it failed, still upload the precomputed buildstocks. So that locally you can just reproduce with the same sampling.

This is helpful only if the sampling is non-deterministic.